modules/cnf-configuring-nrop-on-schedlable-control-planes.adoc

```diff
@@ -0,0 +1,233 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="cnf-configuring-nrop-on-schedulable-cp-nodes_{context}"]
+= Configuring NUMA Resources Operator on schedulable control plane nodes
+
+
+[role="_abstract"]
+This procedure describes how to configure the NUMA Resources Operator (NROP) to manage control plane nodes that a user configures to be schedulable. This is particularly useful in compact clusters where control plane nodes also serve as worker nodes, or in multi-node OpenShift (MNO) clusters where control plane nodes are configured as schedulable to run workloads.
+
+When you configure control plane nodes to be schedulable, you can optionally configure the NUMA Resources Operator to manage them. To do this, define separate `nodeGroups` for the control plane and worker nodes in the NUMAResourcesOperator custom resource (CR)
+
+This configuration allows the NUMA Resources Operator to schedule pods on both control plane and worker nodes, taking into consideration resource availability and NUMA topology. The NUMA Resources Operator treats Control plane nodes like any other schedulable node on the cluster, and they can be included in the `nodeGroups` for Topology-Aware Scheduling (TAS).
```

Comment:
I think this paragraph is already covered in modules/cnf-nrop-support-schedulable-resources.adoc